app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -885,15 +891,21 @@ public void handleHoverEvent(MotionEvent aEvent) {
             updateBorder();
         }
 
-        if (!mActive) {
-            // Do not send touch events to not focused windows.
+        if (!mActive
+                && aEvent.getAction() != MotionEvent.ACTION_SCROLL
+                && aEvent.getAction() != MotionEvent.ACTION_HOVER_MOVE) {
+            // Do not send hover events to not focused windows, except for scrolling and moving events.
             return;
         }
 
         if (aEvent.getAction() == MotionEvent.ACTION_SCROLL) {
             requestFocusFromTouch();
         }
 
+        if (aEvent.getAction() == MotionEvent.ACTION_HOVER_MOVE) {
```

Comment:
I don't think that this is needed. We want to focus the window when the user clicks on it, but otherwise it's fine to just pass the events without making it active.

Note that inactive windows have a tiny window with the URL (left side of the image), and this change disables that.

![Screenshot_com igalia wolvic dev_2024 09 17-15 37 37 920_688](https://github.com/user-attachments/assets/a9375f4f-f6de-468f-9736-2fac3bf338a2)